source/documentors/references/templates/doc_reference_template.rst

```diff
@@ -130,14 +130,14 @@ RST Template
 	External Links
 	--------------
 
-	You can use `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#linking>`_ or `indirect links`_. Indirect links are useful if you want to link to the same thing multiple times, or if the URL is really long and you want the raw RST to read more cleanly.
+	You can use `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference.html#linking>`_ or `indirect links`_. Indirect links are useful if you want to link to the same thing multiple times, or if the URL is really long and you want the raw RST to read more cleanly.
 
-	.. _Indirect links: https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#linking
+	.. _Indirect links: https://docs.openedx.org/en/latest/documentors/references/quick_reference.html#linking
 
 	Internal Links
 	--------------
 
-	Link to :doc:`doc_how_to_template` in the same folder or :doc:`../quick_reference_rst` in a different folder.
+	Link to :doc:`doc_how_to_template` in the same folder or :ref:`Documentation Syntax Reference` in a different folder.
```

Comment:
I really think we should discourage use of `:doc:` or even completely remove the recommendation. We've had so many instances of the docs build failing because of refactorings in this, or other intersphinx-linked, projects.